decidim-core/app/helpers/decidim/sanitize_helper.rb

```diff
@@ -37,7 +37,7 @@ def decidim_sanitize_newsletter(html, options = {})
     end
 
     def decidim_sanitize_editor(html, options = {})
-      content_tag(:div, decidim_sanitize(html, options), class: %w(ql-editor ql-reset-decidim))
```

Comment:
If we're removing `ql-reset-decidim` from the HTML, should we also remove it [from the CSS](https://github.com/decidim/decidim/blob/develop/decidim-core/app/packs/stylesheets/decidim/extras/_quill.scss?plain=1#L15)?

On the other hand, we should also update [this expect](https://github.com/decidim/decidim/blob/develop/decidim-meetings/spec/system/user_edit_meeting_spec.rb?plain=1#L108).

Reply:
It's removed now.